### DISCUSSION TITLE
ScreenLogic And njsPC?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#1016] [Q&A]
For a long time, I was running  both ScreenLogic and njsPc.  

Lately, I've found neither works if the wires from the rpi are connected to the ScreenLogic Protocol adapter.  Maybe that makes sense--unless the rpi outputs driving the prototcol adapter got to high impedance, or they are open drain/collector, then drivers would be "fighting each other".

If that's the case, it's a wonder it worked for so long!

If that IS the case, is there way to reliably have both ScreenLogic AND njsPC both controlling the protocol adapter?

Currently, there  is a box labelled ScreenLogic Wireless Interface Link connected to the protocol adapter.

**Comment by rstrouse** [comment]
That is very odd unless there was a chip failure.  At the transfer rates RS485 should not need termination unless the wiring is very very long.  If the RS485 transceivers are working as expected then simply connecting them on the same bus should allow this to work just fine.

**Reply by davthomaspilot** [reply]
Ok, I'll order a new dongle for the pi and try that.
Thanks
